module/x/gravity/keeper/keeper.go

```diff
@@ -48,11 +50,17 @@ type Keeper struct {
 	ibcTransferKeeper *ibctransferkeeper.Keeper
 	bech32IbcKeeper   *bech32ibckeeper.Keeper
 
+	shutdown bool
+
 	AttestationHandler interface {
 		Handle(sdk.Context, types.Attestation, types.EthereumClaim) error
 	}
 }
 
+func (k *Keeper) Shutdown() {
+	k.shutdown = true
+}
```

Comment:
This will have to be called somewhere.

Reply:
It's there mostly just for testing, will add a comment. I would hardcode it in the NewKeeper

Reply:
Or we can call it in app.go, that's probably cleaner

Reply:
I would prefer to have it as a constructor parameter in that case.

Reply:
could you:
* update the keeper constructor
* in app.go add todo, to set to the shutdown keeper constructor param to `true`